webClient/src/app/browser-panel/browser-panel.component.scss

```diff
@@ -38,4 +38,8 @@ svg {
     min-height: 150px;
 }
 
+.ui-treenode-icon {
+    padding-right: 3px !important;
+    display: table-cell !important;
+  }
```

Comment:
I believe this will mess with the File Explorer styling in the Editor (because it's setting the generic ui-treenode-icon element) so do we need to keep this/what visual function does it serve?

Reply:
I think we can remove this part for sure.